Sources/Core/Extensions/URLQueryParameterEncoder.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  URL+Additions.swift
```

Comment:
Should the file be renamed to URL+Additions?

Reply:
Yea, it probably should. I'll swap it over.